build/build_image.sh

```diff
@@ -30,6 +30,12 @@ if [[ -f "$toplevel/.env" ]]; then
 fi
 
 OS_NAME="${OS_NAME:-ubuntu20.04}"
+
+# force to use ubuntu22.04 image to package milvus
+if [ "$OS_NAME" = "ubuntu20.04" ] || [ -z "$OS_NAME" ]; then
```

Comment:
`-z "$OS_NAME"` condition is not necessary since line#32 has default value. But does not harm anyway.

Reply:
yes, you right